src/main/java/appeng/tile/misc/TileInscriber.java

```diff
@@ -134,15 +132,19 @@ public AECableType getCableConnectionType( final AEPartLocation dir )
 	@TileEvent( TileEventType.WORLD_NBT_WRITE )
 	public void writeToNBT_TileInscriber( final NBTTagCompound data )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00100)